packages/backend/src/routes/shoppingLists.spec.js

```diff
@@ -0,0 +1,142 @@
+import * as request from "supertest";
+import { expect } from "chai";
+import Sequelize from "sequelize";
+const Op = Sequelize.Op;
+
+import {
+  setup,
+  cleanup,
+  randomString,
+  createUser,
+  createSession,
+  createShoppingList,
+} from "../testutils";
+
+import Models from "../models";
+const { ShoppingList } = Models;
+
+describe("shopping Lists", () => {
+  let server;
+  beforeAll(async () => {
+    server = await setup();
+  });
+
+  afterAll(async () => {
+    await cleanup();
+  });
+
+  it("succeeds with no collaborators", async () => {
+    const user = await createUser();
+
+    const session = await createSession(user.id);
+
+    const initialCount = await ShoppingList.count();
+
+    const payload = {
+      title: randomString(20),
+    };
+
+    return request(server)
+      .post("/shoppingLists")
+      .query({ token: session.token })
+      .send(payload)
+      .expect(200)
+      .then(({ body }) =>
+        ShoppingList.findOne({
+          where: {
+            [Op.and]: [payload, { id: body.id, userId: user.id }],
```

Comment:
In sequelize, there's no need for the Op.and here since all `where` conditions must be met (implicitly already and)